setup.py

```diff
@@ -56,7 +56,7 @@
     )],
 
     install_requires=['indy-plenum==1.13.0.dev14',
-                      'importlib-metadata<3.0',
+                      'importlib-metadata==3.10.1',
```

Comment:
Could you add a comment here that says something like:
"Needs to be pinned to because of fpm build issues with a higher version number?
@WadeBarnes @pSchlarb do you have a better suggestion?

Reply:
Not really. Maybe also a cross reference to https://github.com/hyperledger/indy-plenum/blob/ubuntu-20.04-upgrade/build-scripts/ubuntu-2004/build-3rd-parties.sh#L104-L106 .

Reply:
Done.